pinot-segment-local/src/main/java/org/apache/pinot/segment/local/utils/TableConfigUtils.java

```diff
@@ -458,6 +461,18 @@ public static void validateIngestionAggregation(String name) {
         String.format("aggregation function %s must be one of %s", name, SUPPORTED_INGESTION_AGGREGATIONS));
   }
 
+  static void validateDecoder(StreamConfig streamConfig) {
```

Comment:
Should this be called `validateStreamConfig`?

Reply:
I think we can introduce another parent function as we add more validations for streamConfig